server/src/controllers/chess-game.js

```diff
@@ -0,0 +1,8 @@
+import CGUtil from '../utils/chess-game-util';
+
+exports.getRating = (req, res) => {
```

Comment:
we can use es6 here.

Reply:
Also, not sure but controllers does not need to know about req/res objects. It can be a standalone function that returns something wheen asked about something.
And the router can handle req/res part.

Reply:
I read about routes and controllers from [here](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes#Create_the_route-handler_callback_functions), we can change if needed.